cmd/logshare-cli/main.go

```
@@ -87,15 +91,30 @@ func run(conf *config) func(c *cli.Context) error {
 				return err
 			}
 			defer gcsWriter.Close()
-			outputWriter = gcsWriter
+			outputWriters = append(outputWriters, gcsWriter)
+		}
+		if conf.fileDest != "" {
+			destFile, err := os.Create(conf.fileDest)
+			if err != nil {
+				return err
+			}
+			buf := bufio.NewWriter(destFile)
+			defer func() {
+				buf.Flush()
+				destFile.Close()
+			}()
+			outputWriters = append(outputWriters, buf)
+		}
+		if len(outputWriters) == 0 {
+			log.Printf("Warning: No output writers")
```

`no destinations have been set. All output will be silently dropped`